src/components/item-table/item-table-body.tsx

```diff
@@ -63,6 +65,10 @@ export default function ItemTableBody({
       variables: { search: itemGroupSearch },
       skip: !itemGroupSearch,
       onCompleted(data) {
+        try {
+          setHasMore(data.stashSnapshotItemGroupSummaries.hasMore);
+        } catch { }
```

Comment:
Why do we need this try catch here?

Reply:
I had set more as just a function, that was being passed in and wasn't sure if all sources would have to send the function. Now that it's a Dispatch<SetStateAction<boolean>>, I assume the type is going to be more accurate and my concerns for safety are unwarranted. I'll remove the try catch